eth/peerset.go

```diff
@@ -161,15 +85,7 @@ func (ps *peerSet) registerPeer(peer *eth.Peer, ext *snap.Peer) error {
 func (ps *peerSet) unregisterPeer(id string) error {
 	ps.lock.Lock()
 	defer ps.lock.Unlock()
-
-	peer, ok := ps.peers[id]
-	if !ok {
```

Comment:
Why did this go away?

Reply:
That peer is not used, but will keep the return.

Reply:
The peer is not used, but still need to check the peer exists before trying to drop it on L169 :)